apis/nucleus/src/components/ActionsToolbar.jsx

```diff
@@ -120,7 +120,7 @@ function ActionsToolbar({
   const dividerStyle = useMemo(() => ({ margin: theme.spacing(0.5, 0) }));
 
   const popoverAnchorOrigin = {
-    vertical: 15,
+    vertical: 7,
```

Comment:
would it be much harder to only move it when there is no title?

Reply:
No, but we want this to be aligned with the hover menu and selection toolbar in Qlik Sense as well, which complicates it a little bit (the hover menu is coming from within sense-client while the selection toolbar uses this ActionsToolbar in nebula...)